public/components/datasources/home.tsx

```diff
@@ -59,7 +59,7 @@ export const Home = (props: HomeProps) => {
           exact
           path={['/configure/:id+']}
           render={(routerProps) => (
-            <Configure {...commonProps} type={decodeURIComponent(routerProps.match.params.id)} />
+            <Configure {...commonProps} urlType={decodeURIComponent(routerProps.match.params.id)} />
```

Comment:
Naming nit: we get this type from the URL but that doesn't tell us anything about its semantic meaning. If `type` isn't a good enough name we should describe it as `configurationType` or similar.